scenarios/aws/fakeintake.go

```diff
@@ -4,10 +4,11 @@ import (
 	"github.com/DataDog/test-infra-definitions/components/datadog/apps/fakeintake"
 	ddfakeintake "github.com/DataDog/test-infra-definitions/components/datadog/fakeintake"
 	resourcesAws "github.com/DataDog/test-infra-definitions/resources/aws"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/fakeintake/fakeintakeparams"
 )
 
-func NewEcsFakeintake(env resourcesAws.Environment) (*ddfakeintake.ConnectionExporter, error) {
-	fargateInstance, err := fakeintake.NewECSFargateInstance(env)
+func NewEcsFakeintake(env resourcesAws.Environment, options ...fakeintakeparams.Option) (*ddfakeintake.ConnectionExporter, error) {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Exactly what I was thinking ! @pgimalac Can you rebase on this change and add `WithName` option ?

Reply:
Haha I initially did it that way but ended up reverting to something simpler
But that's perfect, I'll rebase ! :)